engine/src/main/java/io/camunda/zeebe/engine/metrics/ProcessEngineMetrics.java

```diff
@@ -36,17 +42,39 @@ public final class ProcessEngineMetrics {
   private static final String ACTION_EVALUATED_FAILED = "evaluated_failed";
   private static final Counter ELEMENT_INSTANCE_EVENTS =
       Counter.build()
-          .namespace("zeebe")
+          .namespace(NAMESPACE)
           .name("element_instance_events_total")
           .help("Number of process element instance events")
-          .labelNames("action", "type", "partition")
+          .labelNames(ACTION_LABEL, TYPE_LABEL, PARTITION_LABEL)
+          .register();
+  private static final String CREATION_MODE_LABEL = "creation_mode";
+  private static final Counter CREATED_PROCESS_INSTANCES =
+      Counter.build()
+          .namespace(NAMESPACE)
+          .name("process_instance_creations_total")
+          .help("Number of created (root) process instances")
+          .labelNames(ORGANIZATION_ID_LABEL, PARTITION_LABEL, CREATION_MODE_LABEL)
           .register();
   private final String partitionIdLabel;
 
   public ProcessEngineMetrics(final int partitionId) {
     partitionIdLabel = String.valueOf(partitionId);
   }
 
+  public void processInstanceCreated(final ProcessInstanceCreationRecord record) {
+    final CreationMode creationMode =
+        record
+                .startInstructions()
+                .iterator()
+                .hasNext() // TODO use isEmpty() which is a commit in another branch
+            ? CreationMode.CREATION_AT_GIVEN_ELEMENT
+            : CreationMode.CREATION_AT_DEFAULT_START_EVENT;
+
+    CREATED_PROCESS_INSTANCES
+        .labels(ORGANIZATION_ID, partitionIdLabel, creationMode.toString())
```

Comment:
:no_good: Please don't use values with big cardinality in labels. https://prometheus.io/docs/practices/naming/#labels 

:thought_balloon: Just as a general thought it feels dirty to couple our product internals with our saas offering.

Reply:
But this is already there in the other metrics. If you want this to change, maybe it is better to open an issue for that and change it for all metrics in this class

Reply:
Wasn't aware sorry:+1:

Reply:
https://github.com/camunda/zeebe/issues/9522